src/procedures.js

```diff
@@ -391,7 +391,7 @@ function makeEditOption(block) {
  * @package
  */
 function deleteProcedureDefCallback(procCode, definitionRoot) {
```

Comment:
Wow the naming of this function is super easy to misread. I thought this was attempting to delete callers but it's actually a /callback/ for deleting the def.

Not actionable.

Reply:
Yeah :/